tendrl/commons/objects/definition/master.yaml

```diff
@@ -103,6 +103,26 @@ namespace.tendrl:
       type: Create
       uuid: 2f94a48a-05d7-408c-b400-e27827f4edef
       version: 1
+    UnmanageCluster:
+      tags:
+        - "tendrl/monitor"
+      atoms:
+        - tendrl.objects.Cluster.atoms.SetClusterUnmanaged
+        - tendrl.objects.Cluster.atoms.StopMonitoringServices
+        - tendrl.objects.Cluster.atoms.StopIntegrationServices
+        - tendrl.objects.Cluster.atoms.DeleteMonitoringDetails
+        - tendrl.objects.Cluster.atoms.DeleteClusterDetails
+      help: "Unmanage a Gluster Cluster"
+      enabled: true
+      inputs:
+        mandatory:
+          - TendrlContext.integration_id
+      pre_run:
+        - tendrl.objects.Cluster.atoms.IsClusterManaged
+      run: tendrl.flows.UnmanageCluster
+      type: Update
+      uuid: 2f94a48a-05d7-408c-b400-e27827f4efed
+      version: 1
```

Comment:
Do we need a post_run to make sure the cluster is un_managed and the services are stopped?

Reply:
May be a post task to make sure cluster is detected back again. Will check this part

Reply:
Please create an issue to track

Reply:
In case of node-agent re-start only we do sds_sync for cluster but in un-manage we are not (re)starting node-agent on storage nodes so removal of tag `indexes/tags/tendrl/integration/{int-id}` should not be done. Also we cannot depend on this to figure out that an un-managed cluster is import ready now.

@r0h4n any other suggestion??

Reply:
I have added a post task for the same and posted below